src/Parser.php

```diff
@@ -26,12 +26,16 @@
  */
 class Parser
 {
+    /** @deprecated Will be removed in v2.0 */
     const INVALID_URI_CHARS = "\x00\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0A\x0B\x0C\x0D\x0E\x0F\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1A\x1B\x1C\x1D\x1E\x1F\x7F";
 
+    /** @deprecated Will be removed in v2.0 */
     const SCHEME_VALID_STARTING_CHARS = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
 
+    /** @deprecated Will be removed in v2.0 */
     const SCHEME_VALID_CHARS = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789+.-';
 
+    /** @deprecated Will be removed in v2.0 */
     const LABEL_VALID_STARTING_CHARS = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
 
     const LOCAL_LINK_PREFIX = '1111111010';
```

Comment:
Same here.

Reply:
Skipped it because it was still used, but I think @nyamsprod said they would be made private, so I'll add them there too.